app/scripts/constants.js

```diff
@@ -84,7 +84,11 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
 
   // This blacklist hides certain kinds from the "Other Resources" page because they are unpersisted, disallowed for most end users, or not supported by openshift but exist in kubernetes
   AVAILABLE_KINDS_BLACKLIST: [],
-
+  // Kill switches for features that are not in tech-preview
+  FEATURE_FLAGS: {
```

Comment:
I'll change for consistency now.  I prob prefer this way for clarity, but def isn't a huge deal.